netsim/ansible/templates/initial/sros.j2

```diff
@@ -80,13 +78,18 @@
 {%  set v6 = l.ipv6|default(False) %}
 {%  set in_base = l.vrf is not defined and (l.vlan is not defined or l.vlan.mode|default('irb')=='route') %}
 
-{%  if l.name is defined %}
+{%  if l.type == "loopback" %}
+{%   set desc = "Loopback" %}
+{%  elif l.name is defined %}
 {%   set desc = l.name|replace('->','~') + " (" + l.role|default('') + ")" %}
-{%  elif l.type|default("") == "stub" %}
+{%  elif l.ifname == "system" %}
+{%   set desc = "System interface" %}
+{%  elif l.type == "stub" %}
 {%   set desc = "Stub interface" %}
 {%  endif %}
 
 {%  set portname = l.parent_ifname if l.type=='vlan_member' else l.ifname %}
+{%  set portname = portname.replace('eth-','') %}
```

Comment:
Not sure I follow why this is necessary - why not remove 'eth-' from ```interface_name```?

Reply:
Your solution created interface names (starting with "i" or "stub") in Jinja2 template. That causes a discrepancy between the device printouts and "netlab report" printouts, and might confuse people using "netlab capture".

This approach makes the interface names consistent, but we have to reverse-engineer portname from interface name.